apps/browser-extension-wallet/src/hooks/__tests__/useWalletManager.test.tsx

```diff
@@ -714,7 +715,8 @@ describe('Testing useWalletManager hook', () => {
           const addAccountProps = {
             wallet: { walletId, type, ...walletProps } as AnyBip32Wallet<Wallet.WalletMetadata, Wallet.AccountMetadata>,
             accountIndex: 0,
-            metadata: { name: 'new account' }
+            metadata: { name: 'new account' },
+            passphrase: Buffer.from('passphrase1')
```

Comment:
It would be better to add it as a property for in-memory object in `walletTypes`, so that it would run this with `passphrase: undefined` for hardware wallet tests
```suggestion
            passphrase
```

Reply:
fixed in 1ee859149027bd295768778e4f04dfb13312ee98